public/views/content/addStudent.html

```diff
@@ -33,7 +33,7 @@
       </div>
 
       <div class="form-group">
-          <button class="btn btn-default" ng-click = "addStudent()" type="submit">Submit</button>
+          <button class="btn btn-default" ng-disabled="!newStudent.archived" ng-click = "addStudent()" type="submit">Submit</button>
```

Comment:
That's what I was going to do initially but it was odd since the way the radio selection works now has two separate input lines.  

I don't see why this doesn't work though? It feels more intuitive to me. Other opinions?